apps/native/app/src/screens/passkey/passkey.tsx

```diff
@@ -79,8 +96,9 @@ export const PasskeyScreen: NavigationFunctionComponent<{
   const intl = useIntl()
   const theme = useTheme()
   const { openBrowser } = useBrowser()
+  const { height } = useWindowDimensions()
+  const isSmallDevice = height < 800
```

Comment:
Just speculating wondering if this is done anywhere else for small or large devices, e.g some hook or util fn

Reply:
Nope, not that I could find at least. I decided when I need this next time I will add it to a util/hook 😄